src/backend/common/queries/team_query.py

```diff
@@ -1,19 +1,22 @@
 from backend.common.models.team import Team
 from backend.common.typed_future import TypedFuture
 from backend.common.queries.database_query import DatabaseQuery
+from backend.common.queries.dict_converters.team_converter import TeamConverter
 from google.cloud import ndb
 from typing import List
 
 
 class TeamQuery(DatabaseQuery[Team]):
+    DICT_CONVERTER = TeamConverter
+
     @ndb.tasklet
     def _query_async(self, team_key: str) -> TypedFuture[Team]:
         team = yield Team.get_by_id_async(team_key)
         raise ndb.Return(team)
 
 
 class TeamListQuery(DatabaseQuery[List[Team]]):
-
+    DICT_CONVERTER = TeamConverter
```

Comment:
you'd probably also need to make a `TeamListConverter` that delegates to `TeamConverter` to make the type checker happy